day_two_guide/topics/proc_restoring-etcd.adoc

```diff
@@ -273,14 +273,17 @@ Before restoring etcd on a static pod:
 * `etcdctl` binaries must be available or, in containerized installations,
 the `rhel7/etcd` container must be available.
 +
-You can obtain etcd by running the following commands:
+You can install the `etcdctl` binary with the etcd package by running the following commands:
 +
 ----
-$ git clone https://github.com/coreos/etcd.git
-$ cd etcd
-$ ./build
+# yum install etcd  
+----
```

Comment:
You need to add a line with a "+" on it, like line 277, so this will render correctly.